functions/create_draft/interactivity_handler.ts

```diff
@@ -73,6 +74,40 @@ export const openDraftEditView: BlockActionHandler<
       });
     }
   }
+  // If the user selects to discard the draft message
+  if (action.selected_option.value == "discard_message_overflow") {
```

Comment:
IMO it makes sense to break both the `edit_message_overflow` and `discard_message_overflow` into separate functions since, once the `action.selected_option.value` is decided, these branches end with exit or error.

It'd make `openDraftEditView` act more as a handler which might make parsing a bit easier, but perhaps not... This is personal preference for sure, so your call on this change!